map_geo.c still uses one deprecated GraphicsMagick call

Though issue #23 was closed when we merged PR #40, it looks like it only addressed Magick calls that were in map_OSM.  This function is deprecated in GraphicsMagick and needs to be replaced with a newer API call.  There is still one such call in map_geo.c.

```

../../Xastir/src/map_geo.c: In function 'draw_geo_image_map':
../../Xastir/src/map_geo.c:2063:9: warning: 'GetIndexes' is deprecated [-Wdeprecated-declarations]
         index_pack = GetIndexes(image);
         ^~~~~~~~~~
In file included from /usr/local/include/GraphicsMagick/magick/api.h:93,
                 from ../../Xastir/src/map_geo.c:137:
/usr/local/include/GraphicsMagick/magick/pixel_cache.h:105:4: note: declared here
   *GetIndexes(const Image *image)
    ^~~~~~~~~~
```

This deprecated call does not get warnings on Ubuntu 16.04 or 14.04, which must have versions of GraphicsMagick that predate the deprecation.  Our OS X Travis jobs are warning about it, though:

```
map_geo.c:2063:22: warning: 'GetIndexes' is deprecated [-Wdeprecated-declarations]
        index_pack = GetIndexes(image);
                     ^
/usr/local/Cellar/graphicsmagick/1.3.30/include/GraphicsMagick/magick/pixel_cache.h:106:5: note: 'GetIndexes' has been explicitly marked deprecated here
    MAGICK_FUNC_DEPRECATED; /* Prefer AccessMutableIndexes() instead */
    ^
/usr/local/Cellar/graphicsmagick/1.3.30/include/GraphicsMagick/magick/common.h:143:56: note: expanded from macro 'MAGICK_FUNC_DEPRECATED'
#      define MAGICK_FUNC_DEPRECATED MAGICK_ATTRIBUTE((__deprecated__))
```

I believe that #40 and #23 discuss the path forward for replacing the GetIndexes calls, or at least #40 does remove some of them and should point the way.  Specifically, commit b071954e put in place the new way of doing things, with ifdefs to allow older versions of GraphicsMagick to work the old way.

--- a/src/map_geo.c
+++ b/src/map_geo.c
@@ -2060,7 +2060,19 @@ void draw_geo_image_map (Widget w,
                             &exception, &da, &pixmap, &gc, screen_width, screen_height))
             return;
 
+#if defined(HAVE_GRAPHICSMAGICK)
+  #if (MagickLibVersion < 0x201702)
         index_pack = GetIndexes(image);
+  #else
+        index_pack = AccessMutableIndexes(image);
+  #endif
+#else
+  #if (MagickLibVersion < 0x0669)
+        index_pack = GetIndexes(image);
+#else
+        index_pack = GetAuthenticIndexQueue(image);
+  #endif
+#endif
         if (image->storage_class == PseudoClass && !index_pack) {
             fprintf(stderr,"PseudoClass && index_pack == NULL!!!");
             if (image)

